js/app.js

```diff
@@ -0,0 +1,100 @@
+		// 1. verbal questions
+
+// a. parameters are the aliases for the values that will be passed through a funciont, arugments are the actual values
```

Comment:
ya -- think of parameters as part of the function definition, and arguments as circumstantial values passed in when the function is called 
i.e. a function "takes" parameters
i.e. when you call a function you "give it" or "pass in" an argument